lib/base/object.hpp

```diff
@@ -57,6 +57,10 @@ extern Value Empty;
 	DECLARE_PTR_TYPEDEFS(klass); \
 	IMPL_TYPE_LOOKUP();
 
+#define RequireNotNull(ptr) RequireNotNullInternal(ptr, #ptr)
```

Comment:
Macros should be capitalised throughout, so `REQUIRE_NOT_NULL` would be correct, wouldn't it?

Reply:
@N-o-X ^